lib/templates/_footer.html

```diff
@@ -4,9 +4,9 @@
     <div class="container-fluid">
         <div class="container">
             <p class="text-center">
-                <span class="no-break">{{^package.isSdk}}package{{/package.isSdk}} {{package.name}}</span>
-                &bull;
-                <span class="no-break">version {{package.version}}</span>
+                <img src="static-assets/favicon.png" alt="Dart" title="Dart"
+                  width="24" height="24" style="margin-right:8px">
+                <span>API docs for {{package.name}}, version {{package.version}}</span>
```

Comment:
If we use bullets, I'd like to make it more clear: the version is the version of the package that we used to generate the docs. I wasn't reading that interpretation strongly before. The bullets, to me, mean that each thing separated is a separate thing. Let's chat about this face to face, I'm confident there's a way to do it so it's easy to read and terse.